src/bot.coffee

```diff
@@ -222,8 +222,9 @@ class SlackBot extends Adapter
     user.room = item.channel
     @receive new ReactionMessage(type, user, reaction, item_user, item, event_ts)
 
+  # Callback for SlackClient.loadUsers()
   loadUsers: (err, res) =>
-    if err || !res.ok
+    if err || !res.members.length
```

Comment:
If the user is cnnecting via `rtm.connect`, there's a valid case where `members` will be empty